fx.go

```diff
@@ -0,0 +1,18 @@
+package orlop
+
+import (
+	"context"
+	"go.uber.org/fx"
+)
+
+type ProvidesFxOptions interface {
+	Options() fx.Option
+}
+
+func FxOptions(o ProvidesFxOptions) fx.Option {
+	return o.Options()
+}
+
+func FxContext(ctx context.Context) fx.Option {
+	return fx.Provide(func() context.Context { return ctx })
+}
```

Comment:
Instead of the functions we call now in server.go, We would call

`app := fx.New(FxOptions(cfg), FxContext(ctx))`

Reply:
Yes